### DISCUSSION TITLE
Can I prevent this very high memory usage?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#8369] [Q&A]
In the screencast below, you can see `htop` running before I open qutebrowser (actually, right after I've closed it; if you think a screencast done right after a system reboot before ever opening qutebrowser, let me know), and right after I open it (with 13 tabs from the saved session, one of which is a YouTube video, as I guess it's important).

The usage of memory skyrokets. 

Is there any way to reduce this memory usage, which doesn't seem to be _so_ extreme right after a reboot, but which takes short time to get to this state?

[![asciicast](https://asciinema.org/a/ayNYWt73yQOXzpPPagLNIbdjF.svg)](https://asciinema.org/a/ayNYWt73yQOXzpPPagLNIbdjF)

**Comment by dpooria** [comment]
I am experiencing the same issue. After a few hours of usage, even if I close all tabs and windows, the browser continues to consume a significant amount of memory (e.g., 3.5 GB). However, the memory usage drops to normal after restarting the qutebrowser. I’m using version 3.2.0 with QtWebEngine 6.8.

**Comment by mosquito-zzzzz** [comment]
indeed, and the memory usage seems to increase with how long you use it, I hope one day it is fixed, because this is holding me back to continue using it

**Comment by edorgeville** [comment]
I find myself running `:restart` once a day, at least it restores my session!

**Comment by lucilanga** [comment]
wi (devtools/inspector) -> Memory -> Collect garbage (the fifth button) - it helps to some degree .. sometimes

**Comment by kachow11** [comment]
hey dudes, has it been fixed? there is another browser named glide-browser but its still experimental if yall wanna check it out
